bot/exts/github/_issues.py

```diff
@@ -0,0 +1,115 @@
+from random import choice
+from typing import Optional
+
+import discord
+from aiohttp import ClientSession
+from bot.constants import Channels, ERROR_REPLIES, Emojis
+from discord import Embed
+from discord.ext import commands
+from loguru import logger
+
+BAD_RESPONSE = {
+    404: "Issue/pull request not located! Please enter a valid number!",
+    403: "Rate limit has been hit! Please try again later!",
+}
+MAX_REQUESTS = 5
+REPO_CHANNEL_MAP = {
+    f"{Channels.dev_reagurk}": "reagurk",
+    f"{Channels.dev_gurkbot}": "gurkbot",
+    f"{Channels.dev_gurklang}": "py-gurklang",
+    f"{Channels.dev_branding}": "branding",
+}
```

Comment:
I don't see why the keys have to be `str`
```suggestion
REPO_CHANNEL_MAP = {
    Channels.dev_reagurk: "reagurk",
    Channels.dev_gurkbot: "gurkbot",
    Channels.dev_gurklang: "py-gurklang",
    Channels.dev_branding: "branding",
}
```
3/4